packages/vite-bundler/package.js

```diff
@@ -2,7 +2,7 @@ Package.describe({
   name: 'vite:bundler',
   version: '0.1.10',
   summary: 'Integrate the Vite.js bundler with Meteor',
-  git: 'https://github.com/Akryum/meteor-vite',
+  git: 'https://github.com/networksforchange/meteor-vite',
```

Comment:
Please make sure that your PR is clean of any references to your personal version of the package.